sources/packages/backend/apps/api/src/services/education-program-offering/custom-validators/has-valid-offering-period-for-funded-days.ts

```diff
@@ -5,78 +5,75 @@ import {
-    const offeringMinDaysAllowedValue = this.getOfferingMinAllowedDays(args);
+    const offeringMinWeeksAllowedValue = this.getOfferingMinAllowedWeeks(args);
     const calculatedStudyBreaksAndWeeks = this.getCalculatedStudyBreaks(
       studyBreaks,
       args,
     );
     return (
-      calculatedStudyBreaksAndWeeks.fundedStudyPeriodDays >=
-        offeringMinDaysAllowedValue &&
-      calculatedStudyBreaksAndWeeks.fundedStudyPeriodDays <=
-        OFFERING_STUDY_PERIOD_MAX_DAYS
+      calculatedStudyBreaksAndWeeks.totalFundedWeeks >=
+      offeringMinWeeksAllowedValue
     );
   }
 
   defaultMessage(args: ValidationArguments) {
-    const offeringMinDaysAllowedValue = this.getOfferingMinAllowedDays(args);
-    return `The funded study amount of days is ineligible for StudentAid BC funding. Your dates must be between ${offeringMinDaysAllowedValue} to ${OFFERING_STUDY_PERIOD_MAX_DAYS} days.`;
+    const offeringMinWeeksAllowedValue = this.getOfferingMinAllowedWeeks(args);
+    return `The funded study amount of days is ineligible for StudentAid BC funding. Your offering must be at least ${offeringMinWeeksAllowedValue} weeks of study or longer to be eligible.`;
   }
 
   /**
-   * Get offering minimum allowed days from args.
+   * Get offering minimum allowed weeks from args.
    * @param args validation arguments.
-   * @returns minimum allowed days.
+   * @returns minimum allowed weeks.
    */
-  private getOfferingMinAllowedDays(args: ValidationArguments): number {
-    const [, , offeringMinDaysAllowed] = args.constraints;
-    return offeringMinDaysAllowed(args.object) as number;
+  private getOfferingMinAllowedWeeks(args: ValidationArguments): number {
+    const [, , offeringMinWeeksAllowed] = args.constraints;
+    return offeringMinWeeksAllowed(args.object) as number;
   }
 }
 
 /**
  * For an offering that contains study breaks, execute the calculation
- * of the funded study period to validate if it matches the mix and max
- * allowed study period amount of days.
+ * of the funded study period to validate if it matches the minimum
+ * allowed study period amount of weeks.
  * @param startPeriodProperty property of the model that identifies the offering start date.
  * @param endPeriodProperty property of the model that identifies the offering end date.
- * @param offeringMinDaysAllowed study period minimum length in number of days.
+ * @param offeringMinWeeksAllowed study period minimum length in number of weeks.
  * @param validationOptions validations options.
  * @returns true if the study period is valid, otherwise, false.
  */
-export function HasValidOfferingPeriodForFundedDays(
+export function HasValidOfferingPeriodForFundedWeeks(
   startPeriodProperty: (targetObject: unknown) => Date | string,
   endPeriodProperty: (targetObject: unknown) => Date | string,
-  offeringMinDaysAllowed: (targetObject: unknown) => number,
+  offeringMinWeeksAllowed: (targetObject: unknown) => number,
```

Comment:
Suggested: `offeringMinFundedWeeksAllowed`

Likewise, please update the comments.